packages/venia-ui/lib/components/Gallery/item.shimmer.js

```diff
@@ -12,7 +12,7 @@ const GalleryItemShimmer = props => {
 
     return (
         <div className={classes.root} aria-live="polite" aria-busy="true">
-            <Shimmer>
+            <Shimmer key="product-image">
```

Comment:
would they not run into duplicates if you have multiple items?

Reply:
Nope! We should be okay since they [only have to be unique within the given parent](https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings) (in this case, the `<div className{classes.root}....>`)

Reply:
Side note, I added this so we can maintain precise targetability. If project's only have <Shimmer> to target, it will be very hard to modify the correct area of the template